lib/widget/screens/home.dart

```diff
@@ -14,24 +14,68 @@ class Home extends StatefulWidget {
 }
 
 class _HomeState extends State<Home> {
+  int _selectedIndex = 0;
+  static const TextStyle optionStyle =
+      TextStyle(fontSize: 30, fontWeight: FontWeight.bold);
+  static const List<Widget> _widgetOptions = <Widget>[
+    PostList(),
+    Text(
+      'Index 1: Creater',
+      style: optionStyle,
+    ),
+    Text(
+      'Index 2: search',
+      style: optionStyle,
+    ),
+    Text(
+      'Index 3: Settings',
+      style: optionStyle,
+    ),
+  ];
+
+  void _onItemTapped(int index) {
+    setState(() {
+      _selectedIndex = index;
+    });
+  }
+
   @override
   Widget build(BuildContext context) {
     return Scaffold(
+      backgroundColor: Color.fromARGB(255, 33, 35, 36),
+      
       appBar: appbar(context),
-      floatingActionButton: FloatingActionButton(
-        onPressed: () async {
-          if (!await isLoggedIn()) {
-            Navigator.push(
-              context,
-              MaterialPageRoute(builder: (context) => Login()),
-            );
-          }
-        },
-        tooltip: 'Publish',
-        elevation: 10,
-        child: const Icon(Icons.add),
+
+      body: Center(
+        child: _widgetOptions.elementAt(_selectedIndex),
       ),
-      body: const PostList(),
+
+      bottomNavigationBar: BottomNavigationBar(
+        type: BottomNavigationBarType.fixed,
+        items: const <BottomNavigationBarItem>[
+          BottomNavigationBarItem(
+            icon: Icon(Icons.home),
+            label: 'Home',
+          ),
+          BottomNavigationBarItem(
+            icon: Icon(Icons.create),
+            label: 'write',
+          ),
+          BottomNavigationBarItem(
+            icon: Icon(Icons.search),
+            label: 'search',
+          ),
+          BottomNavigationBarItem(
+            icon: Icon(Icons.more_vert),
+            label: 'config',
+          ),
```

Comment:
```suggestion
          BottomNavigationBarItem(
            icon: Icon(Icons.create),
            label: 'Write',
          ),
          BottomNavigationBarItem(
            icon: Icon(Icons.search),
            label: 'Search',
          ),
          BottomNavigationBarItem(
            icon: Icon(Icons.more_vert),
            label: 'Settings',
          ),
```
Os nomes dos itens da navigation bar devem começar com letra maiuscula